Cache/Requests.lean

```diff
@@ -103,6 +103,8 @@ def downloadFiles (hashMap : IO.HashMap) (forceDownload : Bool) (parallel : Bool
           msg := msg ++ s!", {failed} failed"
         IO.eprintln msg
       IO.FS.removeFile IO.CURLCFG
+      if success + failed < done then
+        IO.eprintln "Warning: some files were not found in the cache. This usually means that your local checkout of mathlib4 has diverged from upstream."
```

Comment:
```suggestion
        IO.eprintln "Warning: some files were not found in the cache."
        IO.eprintln "This usually means that your local checkout of mathlib4 has diverged from upstream."
        IO.eprintln "If you push your commits to a branch of the mathlib4 repository, CI will build the oleans and they will be available later."
```

Reply:
We could be really really explicit for the poor mathematicians. :-)

Either with the original message or this one, great!

bors d+